Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: retool-wf
 description: A Helm chart for Kubernetes
 type: application
-version: 4.11.18
+version: 5.0.8
```

Comment:
are we jumping ahead to match the main helm version?

Reply:
no this was a mistake because cherry-picking didn't alert me that this would be an issue